committee-steering/governance/annual-reports.md

```diff
@@ -109,15 +113,17 @@ in any area now? What are you doing about it?
 What are some activities that the group does to encourage this? What programs
 are you participating in to grow contributors throughout the contributor ladder?
 - What programs do you participate in for new contributors?
-- Does the group have contributors from multiple companies/affiliations?
+- Does the group have contributors from multiple companies/affiliations? Can end
+ users/companies contribute in some way that they currently are not?
 
 ##### Current initiatives and project health
 - What are initiatives that should be highlighted, lauded, shout outs, that
 your group is proud of? Currently underway? What are some of the longer tail
 projects that your group is working on?
-- What areas does the group need the most help with?
-- Whats the average open days of a PR and Issue in your group? / what metrics does
- your group care about and/or measure?   
+- Year to date KEP work: What's now stable? Beta? Alpha? Road to alpha?
+- What areas and/or subprojects does the group need the most help with?
+- Whats the average open days of a PR and Issue in your group? / What metrics 
+does your group care about and/or measure?   
```

Comment:
This is still somewhat rough to tease out of devstats. the repo group breakdown instead of by sig makes it difficult to pull accurate stats =/ This is something I'm hoping to see change, but it might not be ready for reports.

We may want to either reword this along the lines of "Does your group keep track of open issues/PRs? - If so, can you link any dashboards or data?"

Reply:
I took it off completely but left the last line asking about what metrics they do track and examples; wdyt?

Reply:
that works 👍